<launch>
  <node pkg="jetbot_picam" type="cam_pi.py" name="cam_pi" />
  <group ns="goalpost">
    <node pkg="jetbot_picam" type="color_filter.py" name="goalpost_detector" output="screen">
    </node>
  </group>

  <group ns="ball">
    <node pkg="jetbot_picam" type="color_filter.py" name="ball_detector" output="screen">
      <param name="lower_h" value="3"/>
      <param name="lower_s" value="100"/>
      <param name="lower_v" value="100"/>
      <param name="higher_h" value="20"/>
      <param name="higher_s" value="255"/>
      <param name="higher_v" value="255"/>
    </node>
  </group>
</launch>
